core/controllers/android.py

```diff
@@ -199,13 +243,18 @@ def get(self) -> None:
                     for classroom in config_domain.CLASSROOM_PAGES_DATA.value
                     if classroom['name'] == activity_data['id']
                 ][0]
-                activities[activity_data['id']] = (
+                classroom = (
                     classroom_config_services.get_classroom_by_url_fragment(
```

Comment:
I think we can also remove the old classroom retrieval path while here since it doesn't return usable data, anyway, and the long-term plan is to use the data store version instead of the config property.

Reply:
Done.